src/mutations/user-interaction.mutation.ts

```diff
@@ -0,0 +1,19 @@
+import { gql } from '@apollo/client/core';
+
+export const USER_LIKE_ENTITY = gql`
+    mutation($input: UserInteractionInput!) {
+        userLikeEntity(input: $input)
+    }
+`;
+
+export const USER_UN_lIKE_ENTITY = gql`
```

Comment:
Suggestion: we can named these mutations without separate the word like `USER_UNLIKE_ENTITY` and at the end of the mutations we add "_MUTATION"
`USER_UNLIKE_ENTITY_MUTATION`